yt/data_objects/tests/test_octree.py

```diff
@@ -88,3 +88,53 @@ def test_sph_interpolation_gather():
     )
 
     assert_almost_equal(density.d, answers)
+
+
+def test_octree_properties():
+    """
+    Generate an octree, and test the positions and sizes of the cells.
+    """
+    ds = fake_sph_grid_ds()
+    octree = ds.octree(n_ref=n_ref)
+
+    depth = octree[("index", "depth")]
+    depth_ans = np.array(
+        [
+            0,
+            1,
+            1,
+            1,
+            1,
+            1,
+            1,
+            1,
+            1,
+            2,
+            2,
+            2,
+            2,
+            2,
+            2,
+            2,
+            2,
+        ],
+        dtype=np.int64,
+    )
+    assert_equal(depth, depth_ans)
```

Comment:
I'd suggest 
```suggestion
    depth_ans = np.array([0] + [1]*8 + [2]*8, dtype=np.int64)
    assert_equal(depth, depth_ans)
```

Reply:
Lol! After this I went for that style on the next test with the intention to come back, then my memory failed me.